index.html

```diff
@@ -0,0 +1,184 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="media-style.css" />
+    <title>Anicet's Capstone Project</title>
+</head>
+
+<body>
+    <div class="head-line">
+        <img src="./images/facebook.png" alt="">
+        <img src="./images/twitter.png" alt="">
+        <div>English</div>
+        <div>My page</div>
+        <div>Logout</div>
+    </div>
+    <header>
+        <div class="logo">
+            <a href=""><img src="./images/NFTSbck.jpg" alt="This a NFT logo" /></a>
+        </div>
+
+        <nav>
+            <ul>
+                <li><a href="About.html">About</a></li>
+                <li><a href="">Progran</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsors</a></li>
+                <li><a href="">News</a></li>
+                <li><a href="">Campaign</a></li>
+            </ul>
+        </nav>
+
+    </header>
+    <main>
+        <div class="first-part">
+            <div class="nav-btn">
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+            </div>
+
+            <nav>
+                <ul class="menu-bar">
+                    <li class="close-button">&times;</li>
+                    <li><a href="About.html">About</a></li>
+                    <li>Progran</li>
+                    <li>Join</li>
+                    <li>Sponsors</li>
+                    <li>News</li>
+                    <li>CC campaign</li>
+                </ul>
+            </nav>
+
+            <p class="punchline">"Hello cryptography world"</p>
+            <h1>
+                Start the day with coffee. <br />
+                End the day with NFTS
+            </h1>
+            <p class="punchline2">
+                Anyone who follows the crypto space knows that non-fungible tokens
+                (NFTs) are all the rage of late.
+            </p>
+            <div class="for-date">2021.05.11(THU) ~ 05(FRI)</div>
+            <div class="endline">
+                @ National crypto of Democratic Republic of Congo
+            </div>
+        </div>
+
+        <section class="for-cards">
+            <h2>Main Programs</h2>
+            <div class="card-container">
+                <div class="card00">
+                    <div class="icon"><img src="./images/time-money.png" alt="" /></div>
+                    <div class="card-content">
+                        <h3>time-money</h3>
+                        <p>
+                            Reach out our marketplace, learn more on how to invest in time
+                            and make more experience with us
+                        </p>
+                    </div>
+                </div>
+                <div class="card00">
```

Comment:
- [ ] Kind remainder the second one 👍🏻

Reply:
Thank you for your feedback @Amrhub 

You are really doing a good job, the required changes of the first reviewer for this part was to transform it into article, it's just a reminder, to avoid confusion to other reviewers.

Thank you again and happy coding !!